client/network_status.go

```diff
@@ -0,0 +1,152 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/server"
+	"github.com/skupperproject/skupper/pkg/utils"
+	"time"
+)
+
+func (cli *VanClient) NetworkStatus() ([]*types.SiteInfo, error) {
+
```

Comment:
Suggest checking that skupper router/deployment is running as network status relies on such

Reply:
Good point, I will work on it